src/ext/java/org/opentripplanner/ext/actuator/ActuatorAPI.java

```diff
@@ -39,61 +40,18 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(ActuatorAPI.class);
 
-    public static final PrometheusMeterRegistry prometheusRegistry = new PrometheusMeterRegistry(
+    private static final PrometheusMeterRegistry prometheusRegistry = new PrometheusMeterRegistry(
             PrometheusConfig.DEFAULT
     );
 
+    static {
+        Metrics.globalRegistry.add(prometheusRegistry);
+    }
```

Comment:
Could you add a comment here as to why this is as is?

It would seem a lot cleaner to initialize this in the constructor -- is there a reason that is not possible?

It was only `static` before so that other components could depend on it, but that is not needed anymore.

Reply:
All resources are instantiated per request, and this need to be initialized once. 

I discussed this with @hannesj now, and we decided that we should move it to the OTPApplication and inject it here instead.

Reply:
This was moved to be instantiated once in the OTPApplication, and using the Jersey Context/Binder to inject the instance to the API class.